recipes/rstcheck/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "rstcheck" %}
+{% set version = "3.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 92c4f79256a54270e0402ba16a2f92d0b3c15c8f4410cb9c57127067c215741f
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - rstcheck = rstcheck:main
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - docutils >=0.7
+
+test:
+  imports:
+    - rstcheck
+  commands:
+    - python -m rstcheck --help
```

Can we add ``pip check`` here please?
```suggestion
    - python -m rstcheck --help
    - pip check
  requires:
    - pip
```

---

Sorry, didn't see this in time. Added now!